sandbox/cgroup/cgroup.go

```diff
@@ -242,8 +242,19 @@ func ProcessPathInTrackingCgroup(pid int) (string, error) {
 	// Cgroup entries we're looking for look like this:
 	// 1:name=systemd:/user.slice/user-1000.slice/user@1000.service/tmux.slice/tmux@default.service
 	// 0::/user.slice/user-1000.slice/user@1000.service/tmux.slice/tmux@default.service
+
+	// It seems cgroupv2 can be "dangling" after being mounted and unmounted.
```

Comment:
nice, as you mentioned on IRC it feels like a kernel bug so I think we should also file a LP bug somewhere and mention it here too

Reply:
I'll look for cgroup documentation and kernel source code first. This might be a feature. I agree we need a reference to either the docs or to a bug report.

Reply:
I discussed this with stgraber. The takeways are:

1) controllers are never removed, you must reboot for `0::` to go away
2) there is a kernel bug because the unprivileged container should not have been able to mount it

The bug has been reported in private and has no tracking information.